### ISSUE TITLE
No console bar

### ISSUE THREAD

**Issue by mamiabudat** [stackblitz/webcontainer-core#1345]
### Describe the bug

No console tray at the bottom of the webview when you create a new vanilla javascript project

### Link to the blitz that caused the error

https://stackblitz.com/edit/vitejs-vite-rw8cyj?file=index.html&terminal=dev

### Steps to reproduce

1. Go to stackblitz homepage
2. create a new vanilla javascript project from the template
3. Webview is on the right side, bottom of the webview used to be the console tray.

### Expected behavior

Console tray was useful and handy. Instead of referring to the browser console.

### Parity with Local

- [X] I have [run the project in my local machine](https://github.com/stackblitz/webcontainer-core/blob/main/repro.md#try-locally) and I could not reproduce the issue.

### Platform

```
Browser name  = Chrome
Full version  = 120.0.0.0
Major version = 120
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 64829156,
  "usedJSHeapSize": 61067296,
  "jsHeapSizeLimit": 4294705152
}
Hash = 6f234770
```

### Additional context

It was used to be simple before with firebase extension plugged but now it uses a server from vite

**Comment by Nemikolh** [comment]
Hey @mamiabudat ! :wave: 

We [recently sunsetted our EngineBlock starters](https://blog.stackblitz.com/posts/webcontainers-starters-update/) in favor of having new ones powered by WebContainer and Vite (which provide more flexibility for users that want access to the terminal).

You can see [this comment](https://github.com/stackblitz/webcontainer-core/issues/1336#issuecomment-1957155967) for more information.

**TL;DR**: Open the preview in a separate tab and `Ctrl+Shift+I` (`Option + ⌘ + I` on macOS) -> to see the console.
